store/sqlstore/container.go

```diff
@@ -243,3 +247,17 @@ func (c *Container) DeleteDevice(store *store.Device) error {
 	_, err := c.db.Exec(deleteDeviceQuery, store.ID.String())
 	return err
 }
+
+var (
+	mysqlConflictPattern     = regexp.MustCompile(`ON CONFLICT \([^)]+\) DO UPDATE SET`) // cannot be const
```

Comment:
`ON CONFLICT` is the only statement which is not supported by MariaDB. Everything else is just fine.